modules/learn/pages/clusters-and-availability/xdcr-filtering.adoc

```diff
@@ -163,3 +170,72 @@ image::xdcr/filter-replication-diagram-7.png[,720,align=left]
 
 The new document `airline_8838` is added the source bucket, and is examined by _R1a_.
 A successful match is made, and `airline_8838` is duly replicated to the target bucket.
+
+[#using-deletion-filters]
+== Using Deletion Filters
+
+_Deletion Filters_ control whether the deletion of a document at source causes the deletion of a corresponding replica-document on the target.
+These filters are individually described in xref:manage:manage-xdcr/filter-xdcr-replication.adoc#deletion-filters[Deletion Filters].
+
+By default, any source-document deletion _is_ replicated to the target; resulting in a corresponding target-document deletion.
+Note that such replication is _not_ prevented by the specifying of a filter that is formed with regular and other filtering expressions: such expressions only determine which non-deleted documents are to be replicated.
+Therefore, to ensure that document-deletions are _not_ replicated, _deletion filters_ must specifically be configured.
+
+=== Configuring Deletion-Filters to Protect Data
+
+Appropriate deletion-filter settings protect data.
+However, in certain circumstances, inappropriate deletion-filter settings may cause _loss_ of data.
+For example:
+
+. By means of replication _1_, documents of type _A_ and type _B_ are replicated to the target.
+
+. Replication _1_ is deleted.
+
+. Documents of type _A_ are deleted on the source; with the expectation that they will continue to exist on the target.
+
+. Replication _2_ is created, with the default deletion-filter settings, so as to replicate to the target all future changes on the source to documents of type _B_.
+
+Here, the (incorrectly) expected outcome has been that documents of both type _A_ and type _B_ continue to exist on the target.
+However, since document-deletions are replicated by default, replication _2_ has deleted documents of type _A_ from the target; and the actual outcome is therefore that only documents of type _B_ exist on the target; with documents of type _A_ existing on neither source nor target.
+
+To avoid this outcome, replication _2_ could be created with deletion filters configured to prevent the replication of deletions: the prior deletions of documents of type _A_ from the source would thereby _not_ be replicated to the target.
+Note, however, that this would also prevent the replication of future source-deletions of type _B_ documents: therefore, the creation of an entirely new source collection for type _B_ documents might be required, by means of _migration_, prior to the creation of replication _2_.
```

Comment:
I find this line confusing. I'm not sure what migration has to do specifically with the concept of deletion being talked about here.

Reply:
Perhaps I've misunderstood. Is it just the last sentence that is problematic? ("Note, however," ... etc)

Reply:
Okay, I've removed the line with the reference to migration.